app/notifications/validators.py

```diff
@@ -281,6 +281,16 @@ def service_can_send_to_recipient(send_to, key_type: ApiKeyType, service: Servic
         raise BadRequestError(message=message, status_code=400)
 
 
+def check_service_over_bounce_rate(service_id: str):
+    if current_app.config["FF_BOUNCE_RATE_V1"]:
+        bounce_rate = bounce_rate_client.get_bounce_rate(service_id)
+        if bounce_rate >= current_app.config["BR_CRITICAL_PERCENTAGE"]:
+            # TODO: Bounce Rate V2, raise a BadRequestError when bounce rate meets or exceeds critical threshold
```

Comment:
This makes definitely makes sense, and we may also want to have something further up the call stack to deny notifications before they even get to this point, too.